readme.md

```diff
@@ -0,0 +1,182 @@
+# Robotic Vision Integration Package (RVIP) Documentation
+![Build Status](https://img.shields.io/gitlab/pipeline/cardboardcode/rvip)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![codecov](https://codecov.io/gl/cardboardcode/rvip_ros/branch/master/graph/badge.svg?token=9XFJvJLbpK)](https://codecov.io/gl/cardboardcode/rvip)
+
+## What Is This?
+
+**RVIP** is a ROS package that offers **easy integration of 2D-3D hybrid pose alignment processes** used in robotic vision applications.
+
+By taking in both **2D RGB images** with **3D PointCloud** information, RVIP combines the high effieciency and accuracy of 2D computer vision architectures with the robust data points of depth information.
+
+This is done so in an effort to ensure higher reliability and reduced volatility of object localization results, needed for safer robot-human collaboration spaces.
+
+For a detailed documentation on RVIP, please execute the following commands:
+1. Clone this repository into the `src` folder of your catkin workspace.
+
+> ` $ git clone https://github.com/cardboardcode/rvip.git`
+
+2. Install **rosdoc_lite**.
+
+> `$ sudo apt install ros-melodic-rosdoc-lite`
+
+3. Go to the `rvip` subdirectory and run rosdoc_lite command
+
+> `$ cd rvip && rosdoc_lite .`
+
+4. View the documentation on your browser.
+
+> `$ firefox doc/html/doc/index.html `
+
+## How Does It Work?
+
+The flow of data using RVIP is illustrated in the following diagram.
+
+**Inputs**:
+1. 3D PointCloud (sensor_msgs::PointCloud2)
+2. 2D RGB Camera Image (sensor_msgs::Image)
+
+**Outputs**:
+1. Object Centroid Position, Orientation and Dimensions (visualisation_msgs::MarkerArray,tf::Transform)
```

Comment:
```suggestion
1. Object Centroid Position, Orientation and Dimensions (`visualisation_msgs::MarkerArray`, `tf::Transform`)
```

Reply:
Addressed in [this commit](https://github.com/ros-industrial/rvip/pull/1/commits/e9274a6e3e23f76d7aff505d5f0dcf95797b196f).